client/src/main/java/org/apache/cassandra/sidecar/client/HttpResponse.java

```diff
@@ -62,4 +62,9 @@ default String contentAsString()
      * @return the headers for the response
      */
     Map<String, List<String>> headers();
+
+    /**
+     * @return the server that returns the response
+     */
+    SidecarInstance respondingServer();
```

Comment:
Would it make more sense to call this instance instead?
```suggestion
    SidecarInstance sidecarInstance();
```

Reply:
Yeah. For consistency, `sidecarInstance` fits better and I will update. 
I called it "server" because the meaning of instance is vague.